stable/graylog/templates/ingress.yaml

```diff
@@ -1,6 +1,6 @@
 {{- if .Values.graylog.ingress.enabled -}}
 {{- $extraPaths := .Values.graylog.ingress.extraPaths -}}
-apiVersion: extensions/v1beta1
+apiVersion: networking.k8s.io/v1beta1
```

Comment:
maybe use the capabilities functions to probe for supported apis? so that also older k8s clusters can be supported? https://helm.sh/docs/chart_template_guide/builtin_objects/

Reply:
@davidkarlsen, I have added the changes according to your suggestion. Please verify.